README.md

```diff
@@ -36,11 +36,29 @@ How To (Plugin Developers)
 <dependency>
     <groupId>com.destroystokyo.paper</groupId>
     <artifactId>paper-api</artifactId>
-    <version>1.15.2-R0.1-SNAPSHOT</version>
+    <version>1.16.1-R0.1-SNAPSHOT</version>
```

Comment:
The master branch is not 1.16.1.

Reply:
Fixed, reverted back to 1.15.2-R0.1-SNAPSHOT, my bad.